src/index.js

```diff
@@ -708,7 +696,34 @@ class NearProvider {
                     GAS_AMOUNT,
                     val
                 );
-            } else {
+            } else if (to != "0x0000000000000000000000000000000000000000" && to == txObj.from) {
```

Comment:
we need to add near before we send funds to another evm address. Could this be done in one transaction?